src/open_inwoner/scss/components/Video/Video.scss

```diff
@@ -0,0 +1,14 @@
+.video {
+  &__container {
```

Comment:
Check with @jiromaykin if he wants to fill this in, else delete the empty classes